### PR TITLE
Log In Page and respective components

### LINKED ISSUE #24: Login Page
# Description

Create Login / Start Page and necessary components

# Related to user story

#1 #2

# Notes and resources

https://styled-components.com/docs/advanced#theming
https://www.joshwcomeau.com/react/boop/

# Tasks

- [x] Theme toggler icon
- [x] Log in with Spotify button
- [x] Skip log in and info icon
- [x] Info bubble (show onclick and onhover)
- [x] Import logo svg
- [ ] Add animation to logo
- [ ] Add animation to theme toggler
- [x] Check and adjust responsiveness
- [x] Create page
- [x] Create stories for page and components

### PR DESCRIPTION
**Pull request by ncbecker**
## Description

- Theme Toggle Icon
- Log In Button
- Skip Log In Button with Info Icon

## Related Issue

Fixes #24

### CODE REVIEW

**Comment by ncbecker** [comment]
I still have two CSS issues, maybe someone has an idea?

1) My grid ist not centered on big screens. Max-width should be 834px and then horizontally centered on bigger screens.

Screenshot 2020-12-02 at 10 41 19

2) At certain (small) width, my background is repeated (I have tried no-repeat and cover for the background, but doesn't help). It's not a huge issue as it is fine on small iPhones (problem only appears for even smaller width). But it still annoys me :-)

Screenshot 2020-12-02 at 10 41 49

File: client/src/App.test.js [hunk 1]
```diff
@@ -3,6 +3,6 @@ import App from "./App";
 
 test("renders storybook react link", () => {
   render(<App />);
-  const linkElement = screen.getByText(/To Storybook/i);
+  const linkElement = screen.getByText(/Skip and let me vote!/i);
```
Comment (Sina-Sche) [inline client/src/App.test.js:6]: 🏆 for working with tests 💯

**Review by Sina-Sche — APPROVED** [review]
lgtm :)

**Comment by ncbecker** [comment]
My first CSS issue is solved with `display: flex` (which I thought I have tried 😏), thanks to @Sina-Sche 🙂👍

**Pull request merged**